### PR TITLE
dotCMS/core#23311 Filter forms is not working as expected

### PR DESCRIPTION
**Pull request by zulqarnainvd**
### Proposed Changes
* Add Content Type Search Filter

### Checklist
- [x] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **

### CODE REVIEW

**Comment by zulqarnainvd** [comment]
I added an extra parameter from the front end but currently, API does not support the Content Type filter.

**Review by alfredo-dotcms — APPROVED** [review]

**Review by rjvelazco — APPROVED** [review]

**Review by hmoreras — APPROVED** [review]

**Review by fmontes — CHANGES_REQUESTED** [review]

File: core-web/apps/dotcms-ui/src/app/portlets/dot-containers/container-list/container-list.component.spec.ts [hunk 1]
```diff
@@ -302,6 +312,17 @@ describe('ContainerListComponent', () => {
...
 
+    it('should emit changes in base types selector', () => {
+        fixture.detectChanges();
+        baseTypesSelector = fixture.debugElement.query(
+            By.css('dot-base-type-selector')
+        ).componentInstance;
+        spyOn(comp, 'changeBaseTypeSelector');
+        baseTypesSelector.selected.emit('test');
+
+        expect(comp.changeBaseTypeSelector).toHaveBeenCalledWith('test');
```
[the remark this answers is not in the document]
Comment (zulqarnainvd) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-containers/container-list/container-list.component.spec.ts:323]: Noted

[the remark this answers is not in the document]
Comment (zulqarnainvd) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-containers/container-list/container-list.component.spec.ts]: updated

[the remark this answers is not in the document]
Comment (zulqarnainvd) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-containers/container-list/container-list.component.spec.ts]: updated

**Pull request merged**